python/wpt/importer/__init__.py

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+import sys
+
+from .tree import GitTree, GeckoCommit
```

Comment:
Just curious if there is any particular reason for retaining the 'gecko' terminology in this file (I see few more references to gecko in doc strings below).

Reply:
That's a good question. My thought was to only move the scripts now and then gradually start cleaning them up later. There are also some things that could be shared between the importer and exporter, once these two scripts are in the same module.